### PR TITLE
Proposal to add minimal example

### PR DESCRIPTION
**Pull request by meethariprasad**
Added minimal code that is needed for the people to get started with TF Hub fine tuning, export and reuse exported module.

### CODE REVIEW

**Comment by meethariprasad** [comment]
> @googlebot I signed it!

@googlebot I signed it!

File: docs/tf2_saved_model.md [hunk 1]
```diff
@@ -266,3 +266,56 @@ Weight regularizers on individual layers are saved (with their regularization
 strength coefficients), but weight regularization from within the optimizer
 (like `tf.keras.optimizers.Ftrl.l1_regularization_strength=...)`)
 is lost. Advise consumers of your SavedModel accordingly.
+
+### Re-exporting the fine-tuning result
```
Comment (arnoegw) [inline docs/tf2_saved_model.md]: Hi @meethariprasad, this is high-quality example code, but it is also much, much longer than anything in this documentation. Based on your work, and your verification of this approach, I have created a condensed version https://github.com/tensorflow/hub/pull/445, which IMHO blends in better with the overall flow and verbosity.

Would your suggestion for improvement be met by this condensed version as well?

Comment (meethariprasad) [reply docs/tf2_saved_model.md]: Yes @arnoegw. You have re written it in great way.

Comment (arnoegw) [reply docs/tf2_saved_model.md]: OK, great! I'll proceed with https://github.com/tensorflow/hub/pull/445 and close this.

Thanks for bringing up this gap in our docs, and validating a solution!

**Comment by arnoegw** [comment]
Per review comments, replaced by https://github.com/tensorflow/hub/pull/445

**Pull request closed without merging**